doc/general_documentation/flight_release.md

```diff
@@ -0,0 +1,89 @@
+\page release Creating a Flight Release
+
+# Creating a Flight Release
+
+This page describes the steps required to complete a flight release. It assumes
+that your environment is configured to cross-compile the fsw for the `armhf`
+architecture.
+
+## Create a Release Branch
+
+Release branches will be created on a user's fork. To make a release branch,
+checkout and update develop. Then make a release branch off of develop.
+
+    git checkout develop
+    git fetch upstream
+    git merge upstream/develop
+    git checkout -b release-0.x.x
```

Comment:
The debians have a git hash embedded in them which is already reported by the fingerprinting script. So I think we should not bump the version number unless the version has already been accepted and merged into develop / master.

Reply:
Great! I like that approach. I recommend documenting it with something like the following:

> Note that creating a release branch and specifying a version number indicates that the code on that branch is frozen with respect to further feature changes. However, as you identify bugs during lab testing before deployment to the ISS, you can still apply bug fixes on that release branch without incrementing the version number. Of course, that means that as you iterate, you might end up generating multiple debs with the same version number. For configuration management purposes, when checking whether two deployments used identical debs, you should always compare their git commit hashes rather than their version numbers.